hack/test_errata.py

```diff
@@ -0,0 +1,738 @@
+                    GithubPRMock(GithubUserMock("openshift-bot"), "Enable 4.2.3 in fast channel(s)", [], 4444, "https://errata.devel.redhat.com/advisory/84", "PR_URL4444", "PR_HTML_URL4444")
+                ],
+                {
+                    "errata_id" : 73
+                },
+                2
+            )
+        ]
+
+
+    @patch("github.Github")
+    def test_return_value_is_correct_for_specific_pr(self, Github_mock):
+        """
+        Test retrieving the HTML url of a PR which is related to a specific errata id.
+        """
+        githubrepo = MagicMock()
+        githubtoken = MagicMock()
+        Github_mock.return_value = self.github_object_mock
+        param_list = self.prs_with_html_url_of_expected_pr
+
+        for (prs, message, expected_pr_html_url) in param_list:
+            with self.subTest():
+                self.repo.get_pulls = MagicMock(return_value=prs)
+
+                pr_html_url = errata.lgtm_fast_pr_for_errata(githubrepo, githubtoken, message)
+                self.assertEqual(pr_html_url, expected_pr_html_url)
+
+
+    @patch("github.Github")
+    def test_only_create_issue_on_the_expected_pr(self, Github_mock):
+        """
+        Test creating an issue comment only on the PR which is related to the specific errata id.
+        """
+        githubrepo = MagicMock()
+        githubtoken = MagicMock()
+        Github_mock.return_value = self.github_object_mock
+        param_list = self.prs_with_index_of_expected_pr
+
+        for (prs, message, expected_index_of_pr_to_create_issue) in param_list:
+            self.repo.get_pulls = MagicMock(return_value=prs)
+            errata.lgtm_fast_pr_for_errata(githubrepo, githubtoken, message)
+
+            for index, pr in enumerate(prs):
+                with self.subTest():
+                    if index == expected_index_of_pr_to_create_issue:
+                        pr.create_issue_comment.assert_called_once()
+                    else:
+                        pr.create_issue_comment.assert_not_called()
+
+
+    @patch("github.Github")
+    def test_issue_comment_format(self, Github_mock):
+        """
+        Test the format of the created issue comment on the PR which is related to the specific errata id.
+        """
+        githubrepo = MagicMock()
+        githubtoken = MagicMock()
+        Github_mock.return_value = self.github_object_mock
+        param_list = self.prs_with_index_of_expected_pr
+
+        for (prs, message, expected_index_of_pr_to_create_issue) in param_list:
+            with self.subTest():
+                self.repo.get_pulls = MagicMock(return_value=prs)
+                errata.lgtm_fast_pr_for_errata(githubrepo, githubtoken, message)
+
+                issue_comment = prs[expected_index_of_pr_to_create_issue].create_issue_comment.call_args
+                expected_issue_comment = "Autoapproving PR to fast after the errata has shipped\n/lgtm"
```

Comment:
```
if __name__ == '__main__':
    unittest.main()
```
to actually start tests

Reply:
I was starting the tests using `python -m unittest` in the terminal, so I missed this, thank you.